### PR TITLE
Update another-coffee.md

### PR DESCRIPTION
**Pull request by pjjefferies**
Please review this, @casey-collab

### CODE REVIEW

File: english/coffee/another-coffee/another-coffee.md [hunk 1]
```diff
@@ -4,4 +4,6 @@ You are quite hungry aren't you ?
 
 [Eat a nice pizza slice](../chilli/pizza-with-chilli.md)
 
-[Eat a bagel](../bagel/eat-a-bagel.md)
\ No newline at end of file
+[Eat a bagel](../bagel/eat-a-bagel.md)
+
+[Have another coffee?](../another-coffee.md)
```
Comment (casey-collab) [inline english/coffee/another-coffee/another-coffee.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by pjjefferies** [comment]
fixed link, please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :cherry_blossom:

**Pull request merged**